core.impl/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/impl/jmx/http/ClientRegisterResponse.java

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright 2025 Telefonaktiebolaget LM Ericsson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.ericsson.bss.cassandra.ecchronos.core.impl.jmx.http;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Map;
+
+/**
+ * Class used to construct Jolokia Client Register Response.
+ */
+public final class ClientRegisterResponse
+{
+    // CPD-OFF
+    private Request myRequest;
+    private Value myValue;
+    private int myStatus;
+    private long myTimestamp;
+
+    public Request getRequest()
+    {
+        return myRequest;
+    }
+
+    public void setRequest(final Request request)
+    {
+        myRequest = request;
+    }
+
+    public Value getValue()
+    {
+        return myValue;
+    }
+
+    public void setValue(final Value value)
+    {
+        myValue = value;
+    }
+
+    public int getStatus()
+    {
+        return myStatus;
+    }
+
+    public void setStatus(final int status)
+    {
+        myStatus = status;
+    }
+
+    public long getTimestamp()
+    {
+        return myTimestamp;
+    }
+
+    public void setTimestamp(final long timestamp)
+    {
+        myTimestamp = timestamp;
+    }
+
+    @Override
+    public String toString()
+    {
+        return "NotificationResponse{" + "request=" + myRequest
+                + ", value=" + myValue + ", status=" + myStatus
+                + ", timestamp=" + myTimestamp + '}';
+    }
+
+    public static final class Request
+    {
+        private String myType;
+        private String myCommand;
+
+        // Getters e Setters
+        public String getType()
+        {
+            return myType;
+        }
+
+        public void setType(final String type)
+        {
+            myType = type;
+        }
+
+        public String getCommand()
+        {
+            return myCommand;
+        }
+
+        public void setCommand(final String command)
+        {
+            myCommand = command;
+        }
+
+        @Override
+        public String toString()
+        {
+            return "Request{" + "type='" + myType + '\'' + ", command='" + myCommand + '\'' + '}';
```

Comment:
Why split constants into parts (e.g. '\'' + '}')?

Last comment on this type, but there are more of these below,,,,

Reply:
Auto-generated toString method